starknet_py/tests/e2e/mock/compile_contracts_v2_6.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
```

Comment:
This script has several smaller issues. Did you copy it over from some place, or did you write it from scratch? (I'm wondering if we should just make an issue to fix those, or should this script be fixed in this PR)

Reply:
IMO we should make an issue and change this scripts to use scarb